app/src/main/java/com/cronus/zdone/api/TasksRepository.kt

```diff
@@ -2,22 +2,24 @@ package com.cronus.zdone.api
 
 import com.cronus.zdone.api.model.Task
 import com.cronus.zdone.api.model.TimeProgress
-import com.cronus.zdone.api.model.UpdateTaskResponse
+import com.cronus.zdone.api.model.UpdateDataResponse
```

Comment:
just name change, reflects that UpdateDataResponse is a general response type that applies to basically all `POST`s